README.md

```diff
@@ -35,6 +35,7 @@ No known limitations
 **Version number:**  2.1.0  
```

Comment:
Could we change the double spacing at the end of the version number and subsequent lines to a html
tag instead to avoid the line breaks being removed when white space is removed.